lld/COFF/Symbols.cpp

```diff
@@ -13,6 +13,7 @@
 #include "lld/Common/Memory.h"
 #include "lld/Common/Strings.h"
 #include "llvm/ADT/STLExtras.h"
+#include "llvm/ADT/SmallSet.h"
```

Comment:
The include of `SmallSet.h` is leftover when using `DenseMap`. (And if we don't already do it, we probably should include `DenseMap.h` explicitly somewhere?)

Reply:
I will remove it, we include both `DenseMap.h` and `DenseSet.h` in `InputFiles.h` (included from `Symbols.cpp`).